### PR TITLE
Laravel 9.x Compatibility

### PR DESCRIPTION
**Pull request by laravel-shift**
This is an automated pull request from [Shift](https://laravelshift.com) to update your package code and dependencies to be compatible with Laravel 9.x.

**Before merging**, you need to:

- Checkout the `l9-compatibility` branch
- Review **all** comments for additional changes
- Thoroughly test your package

If you do find an issue, please report it by commenting on this PR to help improve future automation.

### CODE REVIEW

**Comment by laravel-shift** [comment]
:warning: Shift detected a dependency for the `laravel/framework`. Unless you have a special requirement for depending on the entire Laravel framework, it's recommended to add dependencies for the specific `illuminate` packages instead.

**Comment by laravel-shift** [comment]
:warning: Shift detected GitHub Actions which run jobs using a version `matrix`. Shift attempted to [update your configuration](https://github.com/enlightn/enlightn/pull/95/commits/ec5bbab2c905ac1772cb27860f9f97eb2f9d97fa?w=1) for Laravel 9. However, you should review these changes to ensure the desired combination of versions are built for your package.

**Comment by laravel-shift** [comment]
:alembic: **Using this package?** If you would like to help test these changes or believe them to be compatible, you may update your project to reference this branch.

To do so, temporarily add Shift's fork to the `repositories` property of your `composer.json`:

```json
{
    "repositories": [
        {
            "type": "vcs",
            "url": "https://github.com/laravel-shift/enlightn.git"
        }
    ]
}
```

Then update your dependency constraint to reference this branch:

```json
{
    "require": {
        "enlightn/enlightn": "dev-l9-compatibility",
    }
}
```

Finally, run: `composer update`

File: .github/workflows/manual.yaml [hunk 1]
```diff
@@ -13,8 +13,8 @@ jobs:
     strategy:
       fail-fast: false
       matrix:
-        php: [7.3, 7.4, 8.0]
-        laravel: [6.*, 7.*, 8.*]
+        php: [7.3, 7.4, 8.0, 8.0]
```
Comment (jasonmccreary) [inline .github/workflows/manual.yaml:16]: This should not have been duplicated. We'll get it fixed up for next time.

**Comment by crlcu** [comment]
I'm also watching this.

**Comment by crlcu** [comment]
Any other changes are required for this PR to be merged and tagged ?

**Comment by eliasjtg** [comment]
Any update?

**Comment by paras-malhotra** [comment]
L9 support has been added in #100

**Pull request closed without merging**